### ISSUE TITLE
Recent (?) change in ubuntu's install tree temporally seems to break virt-install ubuntu20.04.

### ISSUE THREAD

**Issue by masaki-furuta** [virt-manager/virt-manager#114]
Hello,

I'm sorry if this's been already known issue, but I've just tried install 20.04 from official site with following command and found that their installation tree seems to be changed into 'http://us.archive.ubuntu.com/ubuntu/dists/focal/main/installer-amd64/current/legacy-images/' .
( Debian uses current/images but ubuntu seems to be current/legacy-images instead. )

`
virt-install -d --connect=qemu:///system --name=u20 --memory=2048,maxmemory=4096 --vcpus=4 --autostart --nographics --virt-type kvm --machine pc --cpu=host --os-variant=ubuntu20.04 --controller type=scsi,model=virtio-scsi --disk path=/var/lib/libvirt/images/ubuntu-20.04-live-server-amd64.iso,device=cdrom --disk path=/var/lib/libvirt/images/u20.img,size=20,bus=scsi,cache=unsafe --force --network=bridge:virbr0,model=virtio --location=http://us.archive.ubuntu.com/ubuntu/dists/focal/main/installer-amd64/ --extra-args=console=ttyS0,115200 serial
`

It looks other part perhaps might be okay , as far as I applied following quick dirty patch to virt-manager and install it.
But looks Ubuntu and Debian seems to share this part, so need to fix respectively.

```
--- /usr/share/virt-manager/virtinst/install/urldetect.py-2020-05-27    2020-05-27 15:51:39.833838940 +0900
+++ /usr/share/virt-manager/virtinst/install/urldetect.py       2020-05-27 15:55:26.559861611 +0900
@@ -638,7 +638,7 @@
             return cache.content_regex(mfile, ".*[Dd]ebian.*")
 
         media_type = None
-        if check_manifest("current/images/MANIFEST"):
+        if check_manifest("current/legacy-images/MANIFEST"):
             media_type = "url"
         elif check_manifest("daily/MANIFEST"):
             media_type = "daily"
@@ -692,7 +692,7 @@
         return arch
 
     def _set_url_paths(self):
-        url_prefix = "current/images"
+        url_prefix = "current/legacy-images"
         if self.cache.debian_media_type == "daily":
             url_prefix = "daily"
         elif self.cache.debian_media_type == "mounted_iso_url":
```

Thanks,

**Comment by ptoscano** [comment]
This is due to the "new Ubuntu installer", and will be fixed with #101 or #112.

**Comment by crobinso** [comment]
This is fixed in git now
